broker-core/src/main/java/io/zeebe/broker/clustering/base/topology/TopologyManagerImpl.java

```diff
@@ -225,8 +234,9 @@ private void updatePartitionInfo(Member node) {
     final Properties properties = node.properties();
     final NodeInfo nodeInfo = topology.getMember(Integer.parseInt(node.id().id()));
     for (String p : properties.stringPropertyNames()) {
-      if (p.startsWith("partition")) {
+      if (p.startsWith("partition-")) {
```

Comment:
We should propagate our properties in a type safe way, i.e. a POJO which we de/serialisiere as JSON. This prefix branching is not really optimal.